README.md

```diff
@@ -1,2 +1,16 @@
-# hof-dotfiles
-Common dotfiles used by HOF
+# HOF Dotfiles
+## Common dotfiles used by HOF in a typical build
+
+Adds the following preconfigured dotfiles to the root of your project on `postinstall`.
+
+- `.dockerignore`
+- `.eslintignore`
+- `.eslintrc`
+- `.gitignore`
+- `.jscsrc.json`
+- `.jshintrc`
+- `.travis.yml`
+
+It won't overwrite any current dotfiles of the same name so you can safely install without destroying your own config.
```

Comment:
Will it overwrite a previous version of itself?

If not, then I'm not sure this really gives us any more than a yeoman generator would. (But perhaps this is easier for now!)